app/helpers/active_storage/meta_helper.rb

```diff
@@ -0,0 +1,7 @@
+module ActiveStorage::MetaHelper
+  def active_storage_meta_tags
+    safe_join [
+      tag.meta(name: 'direct-uploads-url', content: rails_direct_uploads_url)
+    ]
```

Comment:
Also, I'm thinking that maybe we should just decorate the specific file input tag with the URL instead. Like we do with other data-url decorations. We'll wrap that into a direct_upload_file_tag anyway.